composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "prestashop/ps_healthcheck",
+  "description": "PrestaShop - Health check",
+  "homepage": "https://github.com/PrestaShop/ps_healthcheck",
+  "license": "AFL-3.0",
+  "version": "0.0.1",
+  "authors": [
+    {
+      "name": "PrestaShop SA",
+      "email": "contact@prestashop.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6.0",
+    "laminas/laminas-diagnostics": "^1.6"
```

Comment:
What is the license of this package ?

Reply:
BSD 3-clause https://packagist.org/packages/laminas/laminas-diagnostics

So it's okay

Reply:
thanks